tests/unit/utils/validation-errors-test.js

```diff
@@ -1,6 +1,13 @@
 import getMessages from 'ember-changeset-validations/utils/get-messages';
 import buildMessage from 'ember-changeset-validations/utils/validation-errors';
 import { module, test } from 'qunit';
+import config from 'ember-get-config';
+import Ember from 'ember';
+
+const {
+  get,
+  set
+} = Ember;
```

Comment:
Possible to import from `@ember/object` path?  Looks like we need to run the module migrator at another time.

Reply:
Ha ha, I've spent quite some time trying to figure out whether to do it or not before submitting the PR!

Here's my train of thoughts: the `result.message` and `context.message` are "exceptions" in a way, in other words opportunities given by the addon to circumvent the standard way of doing things.
I see my PR as "an alternative standard way of doing things", and as such, these properties can still make a lot of sense. I took the path of least resistance and implemented the PR so that it does not short circuit the behavior coming with the presence of `*.message` properties. TBD...

Reply:
Done